FlowCrypt/Controllers/Threads/MessageThread.swift

```diff
@@ -34,7 +34,11 @@ struct MessageThread: Equatable {
     }
 
     var isArchived: Bool {
-        !labels.contains(.inbox) && !labels.contains(.sent)
+        guard let firstMessageLabels = messages.first?.labels else {
+            return false
+        }
+        // Thread is treaded as archived when first message label doesn't contain inbox & sent label
+        return !firstMessageLabels.contains(.inbox) && !firstMessageLabels.contains(.sent)
```

Comment:
I changed `isArchived` logic because it's not correct & same as gmail behavior.
As you can see from below screenshot, this thread is archived one and is shown as archived in gmail (but it wasn't on our app) so I had to update logic to match gmail logic.

![image](https://user-images.githubusercontent.com/77344191/181446948-a71aa3a2-073b-446d-834a-e17be1db955f.png)

Reply:
But then thread will still be shown as `archived` even when new message is received?

Reply:
Ohhh.. Good point!!!
I need to fix above code a bit

Reply:
Fixed